cms/themes/models.py

```diff
@@ -0,0 +1,23 @@
+from typing import TYPE_CHECKING, ClassVar
+
+from django.conf import settings
+from wagtail.admin.panels import FieldPanel
+from wagtail.fields import RichTextField
+
+from cms.core.models import BasePage
+
+if TYPE_CHECKING:
+    from wagtail.admin.panels import Panel
+
+
+class ThemePage(BasePage):  # type: ignore[django-manager-missing]
+    """The Theme page model."""
+
+    template = "templates/pages/theme_page.html"
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "ThemePage"]
+    subpage_types: ClassVar[list[str]] = ["ThemePage", "topics.TopicPage"]
+    page_description = "A theme page, such as 'Economy'."
```

Comment:
If we're not intending to override this on an an instance-specific basis:

```suggestion
    page_description: ClassVar[str] = "A theme page, such as 'Economy'."
```

Reply:
decided to forego this as the description is now a translated string, and I don't like typing `Promise`s :)